mozilla-lockwise/lockwise-ios
# Lockwise keychain launch issues

Fixes #1190 

- Change was made in [PR 1178](https://github.com/mozilla-lockwise/lockwise-ios/pull/1178) that erased entire Keychain instead of specific values when a user logged out. This PR makes a change that specifies when to erase user sync account values and local database values.  This resolves lockwise being unusable after logging out and back in.

To test:
- Launch lockwise with fresh install
- Log in to sync account, verify sync is successful
- Log out of sync account
- Log in to sync account, verify sync is successful
- Force quit app
- Relaunch app and app should continue to sync normally
